modules/exploits/linux/http/eramba_rce.rb

```diff
@@ -0,0 +1,135 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
```

Comment:
I think this exploit is a good candidate for using the `AutoCheck` mixin which will make sure the target appears to be vulnerable before exploiting it.
```suggestion
  include Msf::Exploit::Remote::HttpClient
  prepend Msf::Exploit::Remote::AutoCheck
```